src/main/java/org/elasticsearch/index/query/RangeQueryBuilder.java

```diff
@@ -53,6 +66,12 @@
     public RangeQueryBuilder(String name) {
         this.name = name;
     }
+    
+
+    @Inject
+    public RangeQueryBuilder() {
+        this.name = null;
```

Comment:
Yeah, I think in further refactoring this will end up not being a final method and we won't have to set it for this.